packages/resource-deployment/templates/README.md

```diff
@@ -0,0 +1,55 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+# Services dashboard
+
+Time settings, including the overall time range and granularity of aggregations, can be changed in UTC time button in the top left.
+
+### Dashboard tiles:
+
+-   **Availability**
+    -   (# Availability test failures) / (# Availability test runs)
+    -   Availability tests are run every 5 minutes by the health-monitor-timer-func in web-workers. The test verifies that we can:
```

Comment:
Can you add a line as to how the tests are run (are they run in parallel or are they run in sequence?)
For example if run 1 of the test takes 20 minutes to go past the get result call, will a second test run while test 1 is stuck on the "get result" call ?